js/modules/k6/k6.go

```diff
@@ -158,6 +158,9 @@ func (mi *K6) Check(arg0, checks goja.Value, extras ...goja.Value) (bool, error)
 	if state == nil {
 		return false, ErrCheckInInitContext
 	}
+	if checks == nil {
+		return false, errors.New("No checks provided to `check`") //nolint:stylecheck
```

Comment:
Sorry again :disappointed:  

But do we really need the `nolint` here? In the end, the error looks like this:
```
GoError: No checks provided to `check` at go.k6.io/k6/js/modules/k6.(*K6).Check-fm (native)
```
So it seems like it doesn't matter if the error message starts from the capital, but having no `nolint` code looks more attractive, right?